src/handlers/serenity.rs

```diff
@@ -78,11 +78,6 @@ impl SerenityHandler {
     async fn create_commands(&self, ctx: &Context) -> Vec<Command> {
         Command::set_global_application_commands(&ctx.http, |commands| {
             commands
-                .create_application_command(|command| {
-                    command
-                        .name("autopause")
-                        .description("Toggles whether to pause after a song ends")
-                })
```

Comment:
Why are we deleting autopause?!?!?

Reply:
![ClownClownMemeGIF](https://user-images.githubusercontent.com/5618493/209721934-5869cb48-db6e-4187-905f-f1052a3ae488.gif)